code/lib/cli-storybook/src/automigrate/helpers/cleanLog.ts

```diff
@@ -10,13 +8,3 @@ export const ansiRegex = ({ onlyFirst = false } = {}) => {
 
   return new RegExp(pattern, onlyFirst ? undefined : 'g');
 };
```

Comment:
I think we don't use `ansiRegex` anywhere else, so you can probably remove the entire file.